_backend/base/api/views.py

```diff
@@ -149,6 +146,16 @@ def get_queryset(self):
         return task_model.objects.all()
 
 
-
+# ===================================================
```

Comment:
Is there a need to keep this since you wanted to purge everything that wasn't fully implemented? Or were you planning to implement this later on?